www/pagetpl.php

```diff
@@ -23,8 +23,16 @@ function basePageHeader($title, $focusCtl, $extraOnLoad, $extraHead,
 ?>
 <html>
 <head>
-   <link rel="icon" type="image/png" href="/ifdb-favicon.png">
-   <link rel="shortcut icon" href="/favicon.ico">
+    <link rel="apple-touch-icon" sizes="180x180" href="/favicons/apple-touch-icon.png">
```

Comment:
It's a bit odd to see this here… it's the icon used for "add to home screen" (a2hs) on iOS, supposedly, but it doesn't seem to work when I a2hs on my iPhone 11 running iOS 14.4.2.

Reply:
ohhh, I see the problem. Safari refuses to use basic auth when downloading a favicon.